devtools/etdump/targets.bzl

```diff
@@ -103,9 +149,11 @@ def define_common_targets():
             ],
             deps = [
                 "//executorch/runtime/platform:platform",
+                ":utils",
             ],
             exported_deps = [
                 ":etdump_schema_flatcc",
+                ":utils",
```

Comment:
Doesn't need to be in both deps and exported_deps; since it's internal-only, it'd be better to stay in non-exported deps if possible

Reply:
Removed `:utils` in deps; there're will be some internal issue in `arvr` if only exist in non-exported deps